src/content/admin/private-repositories/github.mdx

```diff
@@ -10,13 +10,35 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import TiersList from '@theme/TiersList';
 
-You can configure Okteto to use a GitHub to authenticate your users as well as give them access to private GitHub repositories.
+You can configure Okteto to give your developers access to private GitHub repositories.
 
 There are several reasons for considering this approach including:
 - Your users can select a repo from a list, instead of having to type the full repository URL.
 - Give everyone on your cluster read-only access to private repositories.
 - Control access to Okteto and the repositories at the Organization level, or per repository.
-- The pipelines use an app-scoped token, instead of a shared user.
+- The pipelines use an app-scoped token, instead of a shared ssh key.
+
+## Configure your GitHub integration
+
+<Tabs
+  defaultValue="self-hosted"
+  values={[
+    { label: 'Self-Hosted', value: 'self-hosted', },
+    { label: 'SaaS', value: 'saas', },
+  ]}
+>
+<TabItem value="self-hosted">
+
+Follow the guide to [configure the GitHub integration(self-hosted/install/guthub.mdx).
```

Comment:
```suggestion
Follow the guide to [configure the GitHub integration](self-hosted/install/guthub.mdx).
```

It is not rendering correctly the link

<img width="694" alt="Screenshot 2024-02-28 at 19 30 49" src="https://github.com/okteto/docs/assets/3510171/1f6c2793-5c8b-4fa7-b52e-dfba8a66eb92">